src/pl/plperl/meson.build

```diff
@@ -8,8 +8,10 @@ plperl_sources = files(
 
 subppdir = run_command(perl, '-e', 'use List::Util qw(first); print first { -r "$_/ExtUtils/xsubpp" } @INC',
   check: true).stdout()
+privlibexpdir = run_command(perl, '-MConfig', '-e',  'print $Config{privlibexp}',
+  check: true).stdout()
 xsubpp = '@0@/ExtUtils/xsubpp'.format(subppdir)
-typemap = '@0@/ExtUtils/typemap'.format(subppdir)
+typemap = '@0@/ExtUtils/typemap'.format(privlibexpdir)
 
```

Comment:
Looks like we already look up privlibexp elsewhere: https://github.com/anarazel/postgres/blob/82a7c37455376173ca9249021a3c63f657091a76/meson.build#L493 and just need to use it?

Reply:
Done.